packages/loot-core/src/shared/transactions.ts

```diff
@@ -164,7 +164,7 @@ export function addSplitTransaction(transactions, id) {
     if (!trans.is_parent) {
       return trans;
     }
```

Comment:
Making `last` into a typed generic in `util.ts` bubbled up type issues to `transactions.ts`. This is the only location in the codebase using `last`, so it seemed most practical to just remove it.